NEWS

```diff
@@ -11,6 +11,241 @@
+#### GO
+ * Added multi-send flag and user memh support in request params
+#### Packaging
+ * Improved dpkg-buildpackage sample command by using explicit arguments
+#### RDMA CORE (IB, ROCE, etc.)
+ * Increased DC initiator index size to 16 bits
+ * Added GGA put_zcopy and get_zcopy implementation
+ * Added DC dcs_hybrid policy
+ * Reduced MLX5/DV stack size consumption
+ * Improved MLX5 log level when allocations are unsupported, for both WC and NC
+ * Added ODP support for verbs and mlx5dv
+ * Added environment variable to enable setting DC initiator capacity
+ * Enabled DDP support on mlx5dv
+ * Added support of cuda-managed memory on IB when ODP is available
+ * Enabled use of implicit ODP with relaxed ordering
+ * Increased DC initiator default count to 32 for performance optimization
+ * Added ConnectX-8 device support to IB
+ * Improved logs by suggesting to increase PF_LOG_BAR_SIZE when UAR allocation fails
+ * Added support for subnet filter list on RoCE GID
+ * Added a reason when a resource's lane is unreachable
+ * Added IB MLX5 as a separate UCX modules with RPM
+ * Improved assertion in DC initiator polling
+ * Added dynamic DC initiator creation
+ * Added support for host memory pipeline through CUDA buffers
+ * Added basic GGA support
+ * Added unit tests for user-mode registration on XGVMI
+ * Added setting IB DEVX atomic_mode based on device capabilities
+ * Removed DC keepalive
+ * Implemented user-mode registration approach for XGVMI
+#### Tests
+ * Reduced stack size in macros
+ * Improve memory allocation retry scheme
+ * Improve code related to CUDA context usage
+ * Added memtype rendezvous tests on 1-byte message
+ * Improved ASAN suppression for memory leak in binutils/bfd
+ * Improved test by caching CUDA device BAR1 available size
+ * Removed the unused sockaddr test application
+ * Added tests for RDMA memory atomic key registration
+ * Improve AM tests by checking capabilities
+#### Tools
+ * Improved performance by increasing window size for put_bw and add get_bw
+ * Added multi-send flag for receive operations in bandwidth benchmarks
+ * Improved ucx_perftest uni-directional test with added fence
+ * Improved ucx_perftest batch helper notes
+#### UCM
+ * Improved log on memory remap failure
+ * Added memory hooks for CUDA memory mapping functions
+#### UCP
+ * Added ACK overhead for ZCOPY protocols event if fast completion flag is set
+ * Improved log with explicit prompt for address v2 configuration on failure
+ * Disabled global VA memory region when error handling is in use
+ * Added interface attribute flag to detect inter-node capability
+ * Enabled using CUDA staging buffers for pipeline protocols by default.
+ * Improved debuggability for remote key size computation
+ * Improve multi-send performance factors computations
+ * Improved rndv/am protocol threshold calculation.
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled marking memory domains as non-cacheable (GDR case)
```

Comment:
pls rephrase

Reply:
Enabled non-cacheable memory domains, activated for GDR